src/main/java/net/atos/client/vrl/VRLClientService.java

```diff
@@ -40,4 +40,11 @@ public CommunicatieKanaal findCommunicatiekanaal(final UUID communicatiekanaalUU
             return null;
         }
     }
+
+    public CommunicatieKanaal findCommunicatiekanaal(final String communicatiekanaalNaam) throws CommunicatiekanaalNotFoundException {
```

Comment:
Een readXXX methode geeft nooit null terug maar gooit een RuntimeException als het betreffende ding niet gevonden wordt.
Een findXXX methode geeft wel null terug als het betreffende ding niet gevonden wordt.

Je hebt hier een combi gemaakt van beide. De CommunicatiekanaalNotFoundException kan wat mij betreft weg, want het voegt niet veel toe deze hier te gooien gezien je er in de ProductAanvraagService niets mee doet behalve loggen. Return null zodat deze methode aan de findXXX conventie van ons voldoet en log hier eventueel dat het kanaal niet bestaat.

Reply:
Opgelost in meest recente commit